app/src/main/java/org/oppia/app/player/audio/CellularDataDialogFragment.kt

```diff
@@ -0,0 +1,55 @@
+package org.oppia.app.player.audio
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.fragment.app.DialogFragment
+import android.widget.CheckBox
+import org.oppia.app.R
+
+/**
+ * DialogFragment that controls user preference for dialog box
+ */
+class CellularDataDialogFragment : DialogFragment() {
+  private var doNotShowAlert: Boolean = false
```

Comment:
Should this be ``doNotShowAgain``?

Reply:
Changed this variable to `doNotShowAgain` and that actually makes more sense now.